docs/location-strategies.adoc

```diff
@@ -179,13 +179,17 @@ public class TestClassOrPageObjectOrPageFragment {
   private List<MyPageFragmentImpl> myListOfPageFragments;
 
   @FindBy(id = "select")
-  private Select select;
+  private GrapheneSelect select;
 }
 ----
 
 As you can see, the `@FindBy` annotation usage is pretty the same as
 with the Selenium 2.
 
+WARNING: Be aware that we use `GrapheneSelect` element instead of default `Select` element.
+The reason behind it is that particular web drivers must implement method `getDomAttribute()`, which was added to the constructor of the `Select` element since Selenium 4.
```

Comment:
```suggestion
The reason behind it is that particular web drivers must implement the method `getDomAttribute()`, which was added to the constructor of the `Select` element since Selenium 4. See the Selenium issues https://github.com/SeleniumHQ/selenium/issues/9895[#9895], https://github.com/SeleniumHQ/selenium/issues/9381[#9381].
```